.bones/tasks/marky-pk33.md

```
@@ -0,0 +1,79 @@
+title: 'PR#41 follow-up: FFI safety hardening — exports.zig bounds check + blob.zig fallibility'
+status: closed
+type: task
+priority: 3
+owner: sethyanow@users.noreply.github.com
+---
+
+## Design
+
+## Goal
+
+Harden FFI boundary with two targeted safety improvements on the Zig side.
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)